TusStorageHandler/minio_client.py

```diff
@@ -0,0 +1,54 @@
+"""Minio Client Configuration Module.
+
+It initializes a Minio client using environment variables
+loaded via dotenv. It also provides a function to create a new Minio
+bucket if it does not already exist.
+
+Usage:
+    - Ensure environment variables are set in a .env file:
+      - MINIO_HOSTNAME
+      - MINIO_PORT
+      - MINIO_ACCESS_KEY
+      - MINIO_SECRET_KEY
+      - MINIO_IS_SECURE (optional, defaults to 'true')
+      - MINIO_BUCKET_NAME
+
+    - Import `minio_client` to access:
+      - `minio_client`: Initialized Minio client instance
+      - `create_bucket()`: Function to create a new bucket if not exists.
+"""
+
+import os
+
+from dotenv import load_dotenv
```

Comment:
I personally think, using dotenv package, and letting python handle env variable is a good idea. I think environment variables should be handled by the "environment", expecting having a `.env` file with creds might be a bad idea. 

What I do is I use either my `bash/zsh{rc}`, export the variable in the shell env or just use a tool that help me manage my env variables, like [direnv](https://direnv.net/). Not asking you to set them up, or change your workflow just think `.env` file might not be ideal imo.

thoughts @uniqueg ??